pkgs/development/compilers/rust/default.nix

```diff
@@ -7,33 +7,36 @@
 let
   rustPlatform = recurseIntoAttrs (makeRustPlatform (callPackage ./bootstrap.nix {}));
   version = "1.22.1";
-in
-rec {
+  cargoVersion = "0.23.0";
+  src = fetchurl {
+      url = "https://static.rust-lang.org/dist/rustc-${version}-src.tar.gz";
+      sha256 = "1lrzzp0nh7s61wgfs2h6ilaqi6iq89f1pd1yaf65l87bssyl4ylb";
+  };
+  # patches that apply to both cargo and rustc
+  commonPatches = [
+    # this patch can be removed when rustc is upgraded to 1.23
+    ./patches/1.22.1-update-vendored-jobserver.patch
```

Comment:
`rustc` is released since 4. January.

Reply:
Yes but I didn't feel confident enough to do a rustc upgrade. If you want to do that instead, go for it. Also note that this bug affects bootstrapping as well, so you'll probably need to use the 1.23 binaries itself for bootstrapping.

Reply:
Those are necessary now anyway as we have rustc 1.24 now.

Reply:
Okay we can close the PR then